src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java

```diff
@@ -53,6 +53,27 @@
  * </li>
  * </ul>
  * <p>
+ *   To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;EmptyCatchBlock&quot;/&gt;
+ * </pre>
+ * <p>
+ *   Example:
+ * </p>
+ * <pre>
+ * class Hello {
```

Comment:
please make examples same as ones below to show difference between configs

Also, description says that `By default check allows empty catch block with any comment inside.` please show this